src/AutoFaker{T}.cs

```diff
@@ -81,10 +94,10 @@ public override List<TType> Generate(int count, string? ruleSets = null)
     {
         Initialize();
 
-        AutoFakerContext context = CreateContext(ruleSets);
-
-        PrepareCreate(context);
-        PrepareFinish(context);
+        CreateGenerateContext(ruleSets);
+        
+        PrepareCreate();
```

Comment:
Could do both these methods as part of class construction since the context is no longer required.